group-generators/helpers/data-providers/atst/queries.ts

```diff
@@ -33,3 +46,33 @@ export const getAttestationsQuery = async (
     `
   );
 };
+
+function parseValue(input: string): WagmiBytes {
```

Comment:
I think you can remove the console.log here :)

Reply:
Oops. You're right! Fixed :)